src/Test.Plugins/Test.Plugins.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup Label="AssemblyAttributes">
+    <AssemblyTitle>Test Plugins Library</AssemblyTitle>
+    <Description>Plugins for testing various extensibility points in the SARIF SDK.</Description>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>net461</TargetFrameworks>
```

Comment:
>net461 [](start = 22, length = 6)

Curious why not reference .netstandard?

Reply:
No particular reason. This little plug-in is intended to be quite distinct from the remainder of the SDK, i.e., it is a surrogate for code produced entirely out of this context. I dropped in net461 just because it was the first framework target name I thought of that I also thought should work. :) 

---
In reply to: [381447919](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381447919) [](ancestors = 381447919)